rook/migration/v1.12/README.md

```diff
@@ -0,0 +1,61 @@
+# Upgrade rook-ceph to v1.12.x
+
+## Prerequisites
+
+- [ ] Check the state of rook-ceph so it is healthy before you begin:
+
+  ```bash
+  helm -n rook-ceph list --all
+  # should show all deployed
+  kubectl -n rook-ceph get jobs
+  # should show all completed
+  kubectl -n rook-ceph get pods
+  # should show all running or completed
+  kubectl -n rook-ceph get cephclusters
+  # should show PHASE = Ready and HEALTH = HEALTH_OK
+  ```
+
+## Apply upgrade - _disruptive_
+
+### **1. Pull chart version**
+
+```console
+git clone --single-branch --branch v1.12.11 https://github.com/rook/rook.git
+sudo cp -rf rook/deploy/charts/* helmfile.d/upstream/
+rm -rf rook
+```
+
+### **2. Pull chart version**
+
+Edit helmfile.yaml to fit the current upgrade.
+
+### **3. Wait for the upgrade to complete**
+
+```console
+helmfile -e workload apply
+```
+
+### **3. Verify cluster health**
+
+Verify the Ceph cluster's health using the [health verification](https://rook.io/docs/rook/v1.12/Upgrade/health-verification/).
```

Comment:
We already do a lot of this in the _"Postrequisites"_ below, so I think this is somewhat redundant.